oe_theme.theme

```diff
@@ -210,153 +210,352 @@ function oe_theme_preprocess_paragraph__oe_links_block(array &$variables): void
-          'label' => $item['label'],
-          'path' => $item['href'],
-          'icon_position' => !empty($item['social_network']) ? 'before' : 'after',
-        ],
-        'icon' => [
-          'path' => $variables['ecl_icon_path'],
-          'type' => !empty($item['social_network']) ? 'branded' : 'ui',
-          'name' => !empty($item['social_network']) ? $item['social_network'] : 'external',
-        ],
-      ];
-    }
+  $variables['processed']['social_media'] = $social_media;
+
+  $institution_links = [];
+  foreach ($corporate_footer['institution_links'] as $link) {
+    $institution_links[] = [
+      'link' => [
+        'label' => $link['label'],
+        'path' => $link['href'],
+      ],
+    ];
   }
+  $variables['processed']['institution_links'] = $institution_links;
 
-  if (!empty($site_specific_footer['other_links'])) {
-    foreach ($site_specific_footer['other_links'] as $item) {
-      $identity['info'][] = [
-        'link' => [
-          'label' => $item['label'],
-          'path' => $item['href'],
-        ],
-      ];
-    }
+  $legal_links = [];
+  foreach ($corporate_footer['legal_links'] as $link) {
+    $legal_links[] = [
+      'link' => [
+        'label' => $link['label'],
+        'path' => $link['href'],
+      ],
+    ];
   }
+  $variables['processed']['legal_links'] = $legal_links;
+  $ecl_branding = theme_get_setting('branding') ?? 'core';
+  $preprocess = '_oe_theme_preprocess_oe_corporate_blocks_eu_footer__' . $ecl_branding;
+  $preprocess($variables);
+}
 
-  return $identity ?? [];
+/**
+ * Process corporate_footer data to match ecl component expected format.
+ *
+ * @param array $variables
+ *   Set of available variables.
+ *
+ * @see hook_preprocess_oe_corporate_blocks_eu_footer().
+ */
+function _oe_theme_preprocess_oe_corporate_blocks_eu_footer__core(array &$variables): void {
+  $variables['corporate_footer']['sections'] = [
+    [
+      'logo' => $variables['processed']['logo'],
+      'description' => $variables['processed']['site_owner'],
+    ],
+    [
+      [
+        'title' => $variables['corporate_footer']['contact_title'],
+        'links' => $variables['processed']['contacts'],
+      ],
+      [
+        'title' => $variables['corporate_footer']['social_media_title'],
+        'links' => $variables['processed']['social_media'],
+      ],
+      [
+        'title' => $variables['corporate_footer']['legal_links_title'],
+        'links' => $variables['processed']['legal_links'],
+      ],
+    ],
+    [
+      'title' => $variables['corporate_footer']['institution_links_title'],
+      'links' => $variables['processed']['institution_links'],
+    ],
+  ];
+}
+
+/**
+ * Process corporate_footer data to match ecl component expected format.
+ *
+ * @param array $variables
+ *   Set of available variables.
+ *
+ * @see hook_preprocess_oe_corporate_blocks_eu_footer().
+ */
+function _oe_theme_preprocess_oe_corporate_blocks_eu_footer__standardised(array &$variables): void {
```

Comment:
Same comments as the one on _oe_theme_preprocess_oe_corporate_blocks_ec_footer__standardised: type is not used on the standarised template.